cognite_toolkit/cdf_tk/load/_base_loaders.py

```diff
@@ -52,13 +52,15 @@ class Loader(ABC):
             when two loaders have the same folder name to differentiate between them. If not set, all files are supported.
         dependencies: A set of loaders that must be loaded before this loader.
         exclude_filetypes: A set of filetypes that should be excluded from the supported filetypes.
+        build_path: The path to the build directory for this particular loader.
     """
 
     filetypes: frozenset[str]
     folder_name: str
     filename_pattern: str = ""
     dependencies: frozenset[type[ResourceLoader]] = frozenset()
     exclude_filetypes: frozenset[str] = frozenset()
+    build_path: Path

```

Comment:
This does not seem like a class variable. Suggest moving it to the `__init__` method.